### PR TITLE
support gn/pad

### PR DESCRIPTION
**Pull request by Jzz24**
support gn/pad

### CODE REVIEW

File: torch_function/__init__.py [hunk 1]
```diff
@@ -37,4 +37,7 @@
 from .SwiGLU import swiglu
 from .Swish import swish
 
+from .GroupNorm import group_norm
+from .Pad import padding
```
Comment (Alcanderian) [inline torch_function/__init__.py:41]: padding?

File: torch_function/Pad.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+    @staticmethod
+    def forward(self, input: torch.Tensor, pad: torch.Tensor,
+                mode: str='constant', value: float=0.):
+        output = torch.nn.functional.pad(input, tuple(pad), mode, value)
+        return output
+
+
+def padding(input: torch.Tensor, pad: Sequence[Union[int, torch.SymInt]],
```
Comment (Alcanderian) [inline torch_function/Pad.py:23]: 不能用pad么

File: torch_function/Pad.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                 mode: str='constant', value: float=0.):
+        output = g.op("opmx::Pad", input, pad,
+                      mode_s = mode,
+                      value_f = value)
+        return output
+
+
+    @staticmethod
+    def forward(self, input: torch.Tensor, pad: torch.Tensor,
+                mode: str='constant', value: float=0.):
```
Comment (Alcanderian) [inline torch_function/Pad.py:18]: value是不是得是一个tensor（scalar就行）

File: docs/operators/Pad.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+### `input`: tensor(T)
+
+N-dimensional tensor
+
+### `pad`: tensor(int64)
+
+m-elements tensor
+
+### `mode`: str
```
Comment (Alcanderian) [inline docs/operators/Pad.md:20]: mode是个attr，放到attr里，按照attr的格式改一下

**Review by Alcanderian — CHANGES_REQUESTED** [review]

File: torch_function/Pad.py (hunk 2 above)
Comment (Jzz24) [reply torch_function/Pad.py:23]: 可以的，主要line23 函数里面有一个pad参数，函数名字和参数名一样，会出问题，容我修改修改
